### DISCUSSION TITLE
Create CMSgov addon for storybook js testing

### DISCUSSION THREAD

**Discussion by karinamzalez** [CMSgov/design-system#1823] [Ideas]
## What
> Create CMSgov addon for storybook js testing 

## Why
- To make testing of individual components that use CMSgov easier

**Comment by pwolfert** [comment]
@karinamzalez, could you elaborate a little more on what such a Storybook add-on would do?

Internally, we use Storybook stories as a basis for our visual regression tests, and we've explored using stories in unit tests but determined that it wouldn't cut down on the size or complexity of our unit tests and would actually add indirection to our code. Would this be a tool for CMSDS users to add to _their_ storybooks?
